website/build.gradle

```diff
@@ -184,3 +202,32 @@ task publishWebsite << {
 
 commitWebsite.dependsOn testWebsite
 publishWebsite.dependsOn commitWebsite
+
+/*
+ * Stages a pull request on GCS
+ * For example:
+ *   ./gradlew :beam-website:stageWebsite -PpullId=${ghprbPullId}
+ */
+task stageWebsite << {
+  assert project.hasProperty('pullId')
+  assert pullId.isInteger()
+
+  def gcs_path = "gs://apache-beam-website-pull-requests/${pullId}"
```

Comment:
Added a -PwebsiteBucket to override it, but didn't make it empty by default because the PullRequestBuilder for jenkins jobs doesn't currently allow jobs to override flags. Perhaps it should.